rust/processor/src/models/fungible_asset_models/v2_fungible_metadata.rs

```diff
@@ -81,7 +63,12 @@ impl FungibleAssetMetadataModel {
             let asset_type = standardize_address(&write_resource.address.to_string());
             if let Some(object_metadata) = object_metadatas.get(&asset_type) {
                 let object = &object_metadata.object.object_core;
-                let is_token_v2 = object_metadata.token.is_some();
+                let (mut maximum_v2, mut supply_v2) = (None, None);
+                let fungible_asset_supply = object_metadata.fungible_asset_supply.as_ref();
+                if let Some(fungible_asset_supply) = fungible_asset_supply {
+                    supply_v2 = Some(fungible_asset_supply.current.clone());
+                    maximum_v2 = fungible_asset_supply.get_maximum();
+                }
```

Comment:
can we rewrite this to 
let (maximum, supply) =  if let some() ...
{(blabla, blabla)} else {(none, none)}